Justfile

```
@@ -215,3 +217,16 @@ bench-node pallet steps="5" repeat="1":
         --steps "{{steps}}" \
         --repeat "{{repeat}}" \
         --template node/benchmark_template.hbs
+
+# Generate the benchmark weights
+generate-weights pallet output_file steps="5" repeat="1":
+    cargo run \
+        -p polka-storage-node -r -F runtime-benchmarks -F testnet -- \
+        benchmark pallet \
+        --wasm-execution=compiled \
+        --pallet "pallet_{{pallet}}" \
+        --extrinsic "*" \
+        --steps "{{steps}}" \
+        --repeat "{{repeat}}" \
+        --template node/benchmark_template.hbs \
+        --output "{{output_file}}"
```

Maybe?
```suggestion
        --output "pallets/{{pallet}}/src/weights.rs"
```
Or so it's separated before you add it
```suggestion
        --output "pallets/{{pallet}}/weights.rs"
```

---

I thought the sp pallet had different placement for the weights but I just noticed there are none there yet so we can place it in `pallets/{{pallet}}/src/weights.rs`